lib/cryptoexchange/exchanges/buyucoin/services/market.rb

```diff
@@ -36,11 +24,11 @@ def adapt(output, market_pair)
           ticker.market    = Buyucoin::Market::NAME
           ticker.ask       = NumericHelper.to_d(output['ask'])
           ticker.bid       = NumericHelper.to_d(output['bid'])
-          ticker.high      = NumericHelper.to_d(output['high_24'])
-          ticker.low       = NumericHelper.to_d(output['low_24'])
-          ticker.change    = NumericHelper.to_d(output['change'])
-          ticker.last      = NumericHelper.to_d(output['last_trade'])
-          ticker.volume    = NumericHelper.to_d(output['vol'])
+          ticker.high      = NumericHelper.to_d(output['h24'])
+          ticker.low       = NumericHelper.to_d(output['l24'])
+          ticker.change    = NumericHelper.to_d(output['c24'])
+          ticker.last      = NumericHelper.to_d(output['LTRate'])
+          ticker.volume    = NumericHelper.to_d(output['LBVol'])
```

Comment:
cross checked with https://trade.buyucoin.com/markets?USDT-BTC it looks like the volume collect refers to `v24`

Reply:
Ah i see